requirements.txt

```diff
@@ -21,3 +21,5 @@ gunicorn==20.0.4
 flask-cors==3.0.10
 flask-swagger-ui==3.20.9
 PyYaml==5.4
+jinja2<3.1.0
+itsdangerous==2.0.1
```

Comment:
Can we not use something newer? Current is `2.1.2`.

Reply:
We probably can, but there are breaking API changes after this version, so we’d need to make some minor updates elsewhere.

That makes sense to do, in my opinion, if we want to view this as a renovation more than a repair job.